 calculators:
  dest_dir: "/home/rachel/ARAIG"
  robot_type: "mobika"
  test_type: "test1_braking"

  test_bagger:
    start_offset: 2 # Wait these many seconds before killing
    stop_offset: 2 # Wait these many seconds before killing
    blacklist:
      - /rosout
    whitelist:
      - /signal/runner/start_robot # topic already published before
      - /signal/runner/test_completed # topic not published yet